packages/runtime/src/widgets/Search.tsx

```diff
@@ -146,6 +148,19 @@ export const Search: React.FC<SearchProps> = ({
     onClearAction?.callback();
   }, [onClearAction]);
 
+  // handle not found content renderer
+  const notFoundContentRenderer = useMemo(() => {
+    const notFoundContent = values?.notFoundContent;
+
+    if (!notFoundContent) {
+      return null;
+    }
+
+    return isString(notFoundContent)
+      ? notFoundContent
+      : EnsembleRuntime.render([unwrapWidget(notFoundContent)]);
+  }, [values?.notFoundContent]);
```

Comment:
what would be default value?

Reply:
@anserwaseem check again please, I added default value